asmcli/commands/create-mesh.sh

```diff
@@ -1,11 +1,17 @@
 create-mesh_subcommand() {
   ### Preparation ###
+
+  # using kubeconfig globally for create-mesh sub-command
+  context_set-option "KUBECONFIG_SUPPLIED" 1
+  # setting same behavior for all environments
+  context_set-option "PLATFORM" "multicloud"
```

Comment:
Same as above

Reply:
i'm going to move the PLATFORM set call more locally but anyway why/to what would you like it to be renamed?